app/models/spree_tax_cloud/tax_computer.rb

```diff
@@ -16,9 +16,11 @@ def compute
     return unless order.tax_cloud_eligible?
     reset_tax_attributes(order)
 
-    transaction = Spree::TaxCloudTransaction.transaction_with_taxcloud(order, NewOrder)
-    response = transaction.lookup
-    logger.debug(response)
+    Timeout.timeout(Spree::Config.try(:taxcloud_timeout) || 9) do
+      transaction = Spree::TaxCloudTransaction.transaction_with_taxcloud(order, NewOrder)
+      response = transaction.lookup
```

Comment:
Adding this limited the syntactical scope of `response` to within this block, so we have to re-do this. 😧